joomla/joomla-cms
# Update mod_breadcrumbs.xml

Pull Request for Issue #9521  .
#### Summary of Changes

set default install value for mod_breadcrumbs caching value to off '0' also caching time to '0' so that it has to be a definite wish to enable it.
#### Testing Instructions

This PR will only have an effect from a clean install on selecting a new breadcrumb menu, going to the advanced settings cache should by default be disabled and caching time 0

I need to find the sample install data and check to see if it's enabled at all please hold off testing at the moment, but I welcome comments.
